src/App.vue

```diff
@@ -182,7 +204,7 @@
                     :selected="false"
                     @click="
                       () => {
-                        mainMenuStep = 1
```

Comment:
It seems to me that the wizard logic got very coupled with the main-menu logic, inside this component file, specially with the `currentConfigMenuComponent` variable.

We can follow with this for now, but it would be important for the future to uncouple than, specially since the wizard will also be used for other parts of the application, like the edit mode and the toolbars.

Reply:
I agree with you.
The idea is to design easy to reuse (but not yet completely generic) components until they need to be reused. This way we save coding time until the component has an real need to be re-instantiated along the app.

Once the wizard is in fact reused, its code will change a bit to be more generic and this coupling will no longer exists.